src/codesyncd/codesyncd.ts

```diff
@@ -102,11 +102,12 @@ export const recallDaemon = (statusBarItem: vscode.StatusBarItem, viaDaemon=true
         // Get updated states
         const canPopulateBuffer = CodeSyncState.get(CODESYNC_STATES.POPULATE_BUFFER_LOCK_ACQUIRED);
         const canSendDiffs = CodeSyncState.get(CODESYNC_STATES.DIFFS_SEND_LOCK_ACQUIRED);
+        const canSendTabs = CodeSyncState.get(CODESYNC_STATES.TABS_SEND_LOCK_ACQUIRED);
```

Comment:
@saadberry I think only DiffSendLock is enough since only 1 IDE instance will be creating socket connection to send both Diffs and Tabs data

Reply:
In general we can call it, canSendSocketData